src/common/interfaces/bank-accounts.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export interface BankAccount {
+  id: string;
+  bank_account_name: string;
+  bank_account_type: string;
+  balance: number;
+}
+
+export interface BankAccDetails {
```

Comment:
Please never use short version for interfaces, nor variables.

No `const abc` or `const b` or `BankAccDetails`. It does save few keystrokes but that's it. 

Please correct to`BankAccountDetails`

Reply:
> Please never use short version for interfaces, nor variables.
> 
> No `const abc` or `const b` or `BankAccDetails`. It does save few keystrokes but that's it.
> 
> Please correct to`BankAccountDetails`

Fixed